packages/utils/src/abbreviate-number/abbreviate-number.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { abbreviateNumber } from './abbreviate-number';
```

Comment:
Migrating `abbreviateNumber` from the `extension` and getting claude to generate these tests.

I'll need to refactor + remove from the `extension`